### PR TITLE
Added support for the *sc_req_item* table - CRUD ticket commands

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16650

## Screenshots
please see in related issue.

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
referenced: https://github.com/demisto/etc/issues/23708

**Comment by bakatzir** [comment]
@liorblob It always returns `1`, regarding of the approval status.
looking in the UI, I do not think it has a state.
Screen Shot 2020-04-21 at 11 32 13

What am I missing?

**Comment by liorblob** [comment]
@bakatzir I think it co-relates with the Stage on the right

**Comment by bakatzir** [comment]
@liorblob it is `fulfillment` when `approval` is: requested/approved/rejected
and `waiting_for_approval` when `approval` is: Not Yet Requested
added it.

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 1]
```diff
@@ -205,12 +213,19 @@ def get_ticket_human_readable(tickets, ticket_type: str) -> list:
...
             mapped_state = state
             if ticket_type in TICKET_STATES:
                 mapped_state = TICKET_STATES[ticket_type].get(state, mapped_state)
             hr['State'] = mapped_state
+        approval = ticket.get('approval', '')
+        if approval:
+            mapped_approval = approval
+            if ticket_type in TICKET_APPROVAL:
+                mapped_approval = TICKET_APPROVAL[ticket_type].get(ticket.get('approval'), mapped_approval)
+                hr['Approval'] = mapped_approval
```
Comment (liorblob) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:228]: this should be indented to the left like in state I think

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py [hunk 2]
```diff
@@ -94,7 +98,7 @@ def test_commands(command, args, response, expected_result, expected_auto_extrac
      EXPECTED_ADD_LINK_HR, True),
     (add_comment_command, {'id': "1234", 'comment': "Nice work!"}, RESPONSE_ADD_COMMENT, EXPECTED_ADD_COMMENT_HR, True),
     (delete_record_command, {'table_name': "alm_asset", 'id': '1234'}, {},
-     'ServiceNow record with ID 1234 was successfully deleted.', True)
+     'ServiceNow record with ID 1234 was successfully deleted.', True),
```
Comment (liorblob) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py:101]: is this ok?

File: Packs/ServiceNow/Integrations/ServiceNowv2/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 ## [Unreleased]
+  - Added support for the *sc_req_item* table when get, create, update, delete ticket commands.
```
Comment (amshamah419) [inline Packs/ServiceNow/Integrations/ServiceNowv2/CHANGELOG.md:2]:
```suggestion
  - Added support for the *sc_req_item* table in the get, create, update, and delete ticket commands.
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
+  - Added the *approval* argument in create and update ticket commands.
```
Comment (amshamah419) [inline Packs/ServiceNow/Integrations/ServiceNowv2/CHANGELOG.md:3]:
```suggestion
  - Added the *approval* argument for the create and update ticket commands.
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py (hunk 1 above)
Comment (bakatzir) [reply Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:229]: If it does, than it is being added to all the other ticket types when CRUDing them, so keeping this indented.

**Pull request merged**